packages/react-scripts/template/README.md

```diff
@@ -476,6 +477,52 @@ import { Navbar, Jumbotron, Button } from 'react-bootstrap';
 
 Now you are ready to use the imported React Bootstrap components within your component hierarchy defined in the render method. Here is an example [`App.js`](https://gist.githubusercontent.com/gaearon/85d8c067f6af1e56277c82d19fd4da7b/raw/6158dd991b67284e9fc8d70b9d973efe87659d72/App.js) redone using React Bootstrap.
 
+## Adding Material UI
+
+[Material Design](https://material.io/guidelines/) is an interface design specification create by Google. It's a popular, and increasingly familiar, look and feel for web applications.
+[material-ui](http://www.material-ui.com/) is a library of React components implementing Material Design.
+To add `material-ui` to your create-react-app project:
+
+```
+npm install --save material-ui react-tap-event-plugin
+```
+
+Add the Roboto font to your project. Material UI components are designed to use it.  
+Add the contents of [this repo](https://github.com/andy-j-d/roboto-font-face) to `src/fonts` (you'll need to create the `fonts` directory).
+Legal information regarding Roboto and other Google Fonts can be [found here](https://developers.google.com/fonts/faq).
+
+In your app container (ie `App.js`), add the following:
+
+```js
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import injectTapEventPlugin from 'react-tap-event-plugin';
```

Comment:
As far as I know, this is a must-have for material-ui, so if we need to avoid it in the docs, I think that's a deal-breaker for adding this. 
http://www.material-ui.com/#/get-started/installation